src/Sdk/DTWebApi/WebApi/RunnerRefreshMessage.cs

```diff
@@ -15,35 +16,32 @@ internal RunnerRefreshMessage()
         {
         }
 
-        public RunnerRefreshMessage(
-            ulong runnerId,
-            String targetVersion,
-            int? timeoutInSeconds = null)
+        [DataMember(Name = "target_version")]
+        public String TargetVersion
         {
-            this.RunnerId = runnerId;
-            this.TimeoutInSeconds = timeoutInSeconds ?? TimeSpan.FromMinutes(60).Seconds;
-            this.TargetVersion = targetVersion;
+            get;
+            set;
         }
 
-        [DataMember]
-        public ulong RunnerId
+        [DataMember(Name = "download_url")]
+        public string DownloadUrl
         {
             get;
-            private set;
+            set;
         }
 
-        [DataMember]
-        public int TimeoutInSeconds
+        [DataMember(Name = "sha256_checksum")]
+        public string HashValue
         {
             get;
-            private set;
+            set;
         }
 
-        [DataMember]
-        public String TargetVersion
+        [DataMember(Name = "os")]
+        public string Platform
```

Comment:
data member name should be same as the properly name in most case, i think.

Reply:
updated